validator_client/src/attestation_service.rs

```diff
@@ -560,12 +567,16 @@ impl<T: SlotClock + 'static, E: EthSpec> AttestationService<T, E> {
                     }
                 }
                 Err(e) => {
-                    for signed_aggregate_and_proof in signed_aggregate_and_proofs {
+                    for (signed_aggregate_and_proof, validator_index) in signed_aggregate_and_proofs
+                        .iter()
+                        .zip(validator_indices.iter())
+                    {
                         let attestation = &signed_aggregate_and_proof.message.aggregate;
                         crit!(
                             log,
                             "Failed to publish attestation";
                             "error" => %e,
+                            "validator_index" => validator_index,
```

Comment:
Here we can also grab the validator index from `signed_aggregate_and_proof.message.aggregator_index`

Reply:
Done.